pkg/registry/storage/csidriver/strategy.go

```diff
@@ -94,6 +97,11 @@ func (csiDriverStrategy) PrepareForUpdate(ctx context.Context, obj, old runtime.
 	if !apiequality.Semantic.DeepEqual(oldCSIDriver.Spec.TokenRequests, newCSIDriver.Spec.TokenRequests) || !apiequality.Semantic.DeepEqual(oldCSIDriver.Spec.RequiresRepublish, newCSIDriver.Spec.RequiresRepublish) {
 		newCSIDriver.Generation = oldCSIDriver.Generation + 1
 	}
+
+	if oldCSIDriver.Spec.SELinuxMountSupported == nil &&
+		!utilfeature.DefaultFeatureGate.Enabled(features.SELinuxMountReadWriteOncePod) {
+		newCSIDriver.Spec.SELinuxMountSupported = nil
```

Comment:
For mutable fields, we should also include the generation increment logic similar to above and update validation.go.

I also need to look up if we allow fields to mutate when the feature is disabled.

Reply:
did you push the update?

Reply:
Oh I see you did but in the "generated files" commit

Reply:
Should we do the feature gate check before the generation update?

Reply:
I cut it as a separate commit